From: Code Review Fix <review@fix>
Subject: [PATCH 03] High: glog* 별칭을 mgmt.log 경로로 업데이트

mgmt_log_flush.sh가 mgmt.log에 flush하도록 수정된 것에 맞춰
glog* 별칭도 gmgmt.log → mgmt.log 로 통일.

주의: gmgmt.log가 다른 경로에서 여전히 생성된다면,
      glog* 별칭의 목적(proc 기반 드라이버 로그 보기)에 맞게
      별칭 이름을 유지하되 파일명만 수정.

---
 dist/wlan/etc/profile.d/00-cantops.sh | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/dist/wlan/etc/profile.d/00-cantops.sh b/dist/wlan/etc/profile.d/00-cantops.sh
index 0000001..0000002 100644
--- a/dist/wlan/etc/profile.d/00-cantops.sh
+++ b/dist/wlan/etc/profile.d/00-cantops.sh
@@ -86,8 +86,8 @@ alias mlogf1='tail -f $CAPDIR1/mgmt.log'

-alias glog='cat $CAPDIR0/gmgmt.log'
-alias glogg='cat $CAPDIR0/gmgmt.log | grep -i $1 -a'
-alias glogt='cat $CAPDIR0/gmgmt.log |tail -n $1'
-alias glogh='cat $CAPDIR0/gmgmt.log |head -n $1'
-alias glogf='tail -f $CAPDIR0/gmgmt.log'
+alias glog='cat $CAPDIR0/mgmt.log'
+alias glogg='cat $CAPDIR0/mgmt.log | grep -i $1 -a'
+alias glogt='cat $CAPDIR0/mgmt.log |tail -n $1'
+alias glogh='cat $CAPDIR0/mgmt.log |head -n $1'
+alias glogf='tail -f $CAPDIR0/mgmt.log'
